cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/ProgramScheduleStoreDataset.java

```diff
@@ -374,6 +378,57 @@ public void deleteSchedules(Iterable<? extends ScheduleId> scheduleIds) throws N
     return deleted;
   }
 
+  /**
+   * Update all schedules that can be triggered by the given deleted program. Schedules will be removed if they
+   * contain single {@link co.cask.cdap.internal.app.runtime.schedule.trigger.ProgramStatusTrigger}. Schedules with
```

Comment:
we usually import the package instead of having the full classname in the javadoc.